crates/bevy_geometry/src/lib.rs

```diff
@@ -1,28 +1,183 @@
-use bevy_math::{Quat, Vec3};
+use bevy_math::*;
+use bevy_reflect::Reflect;
+//use bevy_transform::components::GlobalTransform;
+use std::error::Error;
+use std::fmt;
 
-pub trait Primitive3d {}
+pub trait Primitive3d {
+    /*
+    /// Returns true if this primitive is on the outside (normal direction) of the supplied
+    fn outside_plane(
```

Comment:
I see how this might be confusing. It's checking if a 3d objects is fully on the outer side of the plane - i.e. fully contained by the halfspace in the direction of the plane's normal. I'll poke around and see if I can find a standard name.